integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -109,38 +110,14 @@ spec:
             - name: alluxio-fuse-mount
               mountPath: /alluxio-fuse
               mountPropagation: Bidirectional
-            {{- if eq .Values.worker.shortCircuitPolicy "uuid" }}
+            {{- if $shortCircuitEnabled}}
+              {{- if eq .Values.shortCircuit.policy "uuid" }}
             - name: alluxio-domain
               mountPath: /opt/domain
-            {{- end }}
-            {{- if eq .Values.worker.shortCircuitPolicy "local" }}
-              {{- if .Values.tieredstore }} 
-                {{- if .Values.tieredstore.levels }} 
-                  {{- range .Values.tieredstore.levels }}
-                    {{- if .mediumtype }}
-                      {{- if contains "," .mediumtype }}
-            {{- $type := .type }}
-            {{- $path := .path }}
-            {{- $split := split "," .mediumtype }}
-                        {{- range $key, $val := $split }} 
-                          {{- if eq $type "hostPath"}}
-            - mountPath:  {{ index ($path | split ",") $key }}
-              name: {{ $val | lower }}-{{ $key | replace "_" "" }}
-                          {{- end}}
-                        {{- end}}
-                      {{- else}}
-                        {{- if eq .type "hostPath"}}
-            - name: {{ .mediumtype | replace "," "-" | lower }}
-              mountPath: {{ .path }}
-                        {{- else }}
-            # report error and exit
-            {{ .name }} with {{ .type }} is not supported in shortCircuitPolicy local
-                        {{- end }} 
-                      {{- end }}
-                    {{- end}} 
-                  {{- end }} 
-                {{- end }} 
-              {{- end }} 
+              {{- end }}
+              {{- if eq .Values.shortCircuit.policy "local" }}
+{{- include "alluxio.worker.tieredstoreVolumeMounts" . }}
+              {{- end }}
```

Comment:
In my understanding `local` means directly use tiered storage so this should use the template.

In the existing template there's only `hostPath`, I believe this was not the intention.